index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <title>Kwita Izina</title>
+</head>
+<body>
+
+    <div class="summit">
+        <header class="index-header">
+            <div class="menubar">
+                <img src="image/menumob.png" class="menumob" alt="Mobile menu icon"/>
+            </div>
+            <div class="deskheader">
```

Comment:
- [ ] [REQUIRED] Please, use `<section>` element instead of `<div>` elements for semantic purposes.
- [ ] [REQUIRED] Please, make sure that you organize your sections into 3 main ones (section 1: intro, section 2: main program, section 3: speakers)

Reply:
Done

Reply:
Done